_posts/2022-12-01-athena.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+title:  "Trino delivers for Amazon Athena"
+author: Manfred Moser, Martin Traverso
+image: /assets/trino-light.png
+excerpt_separator: <!--more-->
+---
+
+It is time to reach out to another large group of Trino users. The release of
+the new engine version for [Amazon Athena](https://aws.amazon.com/athena)
+upgrades Athena to a recent version of Trino from a rather old version. This
+update brings a ton of improvements from the Trino project to the users of the
+popular cloud-based query service.
+
+<!--more-->
+
+## Shared history
+
+Amazon Athena and Trino share a long history. From the beginning of Athena the
+query engine under the hood was Trino, then still called Presto. Athena created
+a low-maintenance, powerful access mode to your data in S3 and beyond. It
+combined the performance and features of Trino, with the convenience of a cloud
+service, and therefore enabled new users and use cases. You could take advantage
+of Trino, without needing a team of experts to deploy and operate a Trino
+cluster for your organization. In fact, we wrote about this in the first edition
+of **Trino: The Definitive Guide**. There is also a section in the [new second
+edition]({% post_url 2022-10-03-the-definitive-guide-2 %}), that you can get for
+[free from Starburst](https://www.starburst.io/info/oreilly-trino-guide/).
+
+## Time flies
+
+Since the initial release however, time has not stood still. In fact, the Trino
```

Comment:
```suggestion
But since the initial release of Athena, time has not stood still. In fact, the Trino
```
I know some people teach that we shouldn't use conjunctions at the start of a sentence, but it's a rule that's very ok to break to make things flow better, especially in something like a blog post (as opposed to a formal technical guide).